config/rootfs/debos/overlays/cros-flash/opt/chromeos/install-modules

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -xe
+
+install_modules()
+{
+    local mount_dir=$(basename "$IMAGE_MOUNTPOINT")
+    local root_path=$(dirname "$IMAGE_MOUNTPOINT")
+    local modules_tarball=$(basename "$MODULES_URL")
+
+    echo "Downloading modules from $MODULES_URL..."
+    mkdir -p "$IMAGE_MOUNTPOINT"
+    cd "$root_path"
+    wget --no-check-certificate "$MODULES_URL"
```

Comment:
Do we need the --no-check-certificate if we have the ca-certificates package installed in the image?

Reply:
No, good point.  I'm dropping that now and also fixing other issues with flash and install-modules.